Fixes these GCC errors:

In file included from tabs.c:27:
mpfi-tests.h:49:33: error: unknown type name 'intmax_t'
   49 | typedef int (*JS_fun)  (mpfi_t, intmax_t);
      |                                 ^~~~~~~~
mpfi-tests.h:50:33: error: unknown type name 'uintmax_t'
   50 | typedef int (*JU_fun)  (mpfi_t, uintmax_t);
      |                                 ^~~~~~~~~
mpfi-tests.h:96:3: error: unknown type name 'JS_fun'
   96 |   JS_fun  JS;      /* output: mpfi_t, input: intmax_t */
      |   ^~~~~~
mpfi-tests.h:97:3: error: unknown type name 'JU_fun'
   97 |   JU_fun  JU;      /* output: mpfi_t, input: uintmax_t */
      |   ^~~~~~
mpfi-tests.h:248:45: error: unknown type name 'JU_fun'; did you mean 'IU_fun'?
  248 | void mpfi_fun_init_JU   (mpfi_function_ptr, JU_fun, NULL_fun);
      |                                             ^~~~~~
      |                                             IU_fun
mpfi-tests.h:249:45: error: unknown type name 'JS_fun'; did you mean 'IS_fun'?
  249 | void mpfi_fun_init_JS   (mpfi_function_ptr, JS_fun, NULL_fun);
      |                                             ^~~~~~
      |                                             IS_fun

--- a/tests/mpfi-tests.h	2022-03-14 10:38:11.000000000 -0600
+++ b/tests/mpfi-tests.h	2022-03-28 16:29:25.756375645 -0600
@@ -28,6 +28,7 @@ MA 02110-1301, USA. */
 #define __MPFI_TESTS_H__
 
 #include <stdlib.h>
+#include <stdint.h>
 
 #include "mpfi_io.h"
 #include "mpfi-impl.h"
